documentation/src/main/markdown/troubleshooting/solutions.md

```diff
@@ -208,7 +205,7 @@ Caused by: java.lang.IllegalStateException: 'config' is not a directory
 
 ### Possible cause
 
-[solution_name] 7 upgraded to a newer version of Spring Boot (currently: 2.4.5) which by default looks for a subdirectory in the current working directory called "config".
+[solution_name] 8 upgraded to a newer version of Spring Boot (currently: 2.6.6) which by default looks for a subdirectory in the current working directory called "config".
```

Comment:
I made the change due to upgrading the version of Spring Boot again in 8 but if the issue itself does not exist in 8, please confirm, then I'd also say remove it.

Reply:
Yeah, I like the idea of removing the whole symptom/cause/solution. Since users should never upgrade from 6 directly to 8, they should not run into it upgrading to 8.

Reply:
Done. Thank you.